media/javascript/elastic-search.js

```diff
@@ -0,0 +1,17 @@
+// Qs is defined on readthedocs/templates/docsitalia/overrides/search/elastic_search.html on bottom
```

Comment:
questo codice possiamo includerlo in un nostro file js caricato dal template di cui facciamo l'override?

Reply:
@pauloxnet questo non mi risulta risolto, o sbaglio?

Reply:
scusate ho visto adesso che è un file "nostro", non potrebbe essere messo in `readthedocs/docsitalia/static/js`? così è più chiaro il suo ruolo

Reply:
@yakky assolutamente si, attualmente però `readthedocs/docsitalia/static` viene usata da gulp, ci finiscono dentro i files compilati e non vorrei che qualcuno dei nostri venga sovrascritto chissà per quale mistero.
Posso metterli dentro `readthedocs/docsitalia/static-src` e li diamo in pasto a gulp come succede per `readthedocs/docsitalia/static-src/projects/import.js` e `readthedocs/docsitalia/static-src/projects/tools.js`

Reply:
@carmelocatalfamo sarebbe :top: almeno sono tutti gestiti allo stesso modo

RTD già fa confusione di suo, cerchiamo di risparmiarci mal di testa :)